Makefile

```
@@ -62,13 +67,39 @@ wallet-webview:
 
 # Compile only the webview bursa-wallet binary, assuming the web bundle has
 # already been built into the //go:embed dist target. Honors GOOS/GOARCH.
-wallet-binary-webview:
-	cd ui && CGO_ENABLED=1 go build \
+wallet-binary-webview: webkit-shim
```

checkmake (0.3.2)

[warning] Target body for "wallet-binary-webview" exceeds allowed length of 5 lines (7).

(maxbodylength)